pai-management/doc/paictl.md

```diff
@@ -4,226 +4,214 @@ A tool to manage your pai cluster.
 
 ```
-paictl.py machine remove -p /path/to/cluster-configuration/dir -l node-list.yaml
+paictl.py machine remove -p /path/to/cluster-configuration/dir -l machine-list.yaml
 ```
 
-1) You will have to pass [a node list](#Machine_Nodelist_Example) to this command
-
-
-#### An example of node-list <a name="Machine_Nodelist_Example"></a>
-
-An example of machine in the machine list
-```yaml
-
-machine-list:
-
-    - hostname: hostname (echo `hostname`)
-      hostip: IP
-      machine-type: D8SV3
-      etcdid: etcdid1
-      #sshport: PORT (Optional)
-      #username: username (Optional)
-      #password: password (Optional)
-      k8s-role: master
-      dashboard: "true"
-      zkid: "1"
-      pai-master: "true
-
-    - hostname: hostname
-      hostip: IP
-      machine-type: NC24R
-      #sshport: PORT (Optional)
-      #username: username (Optional)
-      #password: password (Optional)
-      k8s-role: worker
-      pai-worker: "true"
-
-```
+- See an example of the machine list [here](#Machine_Nodelist_Example).
 
-## Maintain your service <a name="Service"></a>
+## Maintain infrastructure services <a name="Service"></a>
 
-#### Start service in the cluster <a name="Service_Start"></a>
+### Start service(s) <a name="Service_Start"></a>
 
 ```
 paictl.py service start -p /path/to/cluster-configuration/dir [ -n service-name ]
 ```
 
-1) Starting all service in your cluster.
-2) If the option -n is set, only the target service will be deployed.
+1) Start all services by default.
+2) If the option `-n` is set, only the specified service will be started.
 
-
-#### Stop service in the cluster <a name="Service_Stop"></a>
+### Stop service(s) <a name="Service_Stop"></a>
 
 ```
 paictl.py service stop -p /path/to/cluster-configuration/dir [ -n service-name ]
 ```
 
-1) Stop all service in your cluster.
-2) If the option -n is set, only the target service will be stopped.
-
+- Stop all services by default.
+- If the option `-n` is set, only the specified service will be stopped.
 
-#### Delete (Stop and clean Data) Service in the cluster <a name="Service_Delete"></a>
+### Delete service(s) <a name="Service_Delete"></a>
 
 ```
 paictl.py service delete -p /path/to/cluster-configuration/dir [ -n service-name ]
 ```
 
-1) Firstly, it will stop all service on your cluster.
-2) Secondly, it will delete all service data which has been persistenced on the local host such as hdfs, yarn, zookeeper etc.
-3) If the option -n is set, only the target service will be stopped and deleted.
+- 'Delete' a service means to stop that service and then delete all of its persisted data in HDFS, Yarn, ZooKeeper, etc. 
+- Delete all services by default.
+- If the option `-n` is set, only the specified service will be deleted.
 
-#### Upgrade service in the cluster <a name="Service_Upgrade"></a>
+### Upgrade service(s) <a name="Service_Upgrade"></a>
 
 ```
 paictl.py service upgrade -p /path/to/cluster-configuration/dir [ -n service-name ]
```

Comment:
@ydye : Will user use this command explicitly? If not, we'd better remove it here and then integrate its functionality into other commands.

Reply:
I want to replace it here. 
1) To refresh the node-label when new machine added.